### DISCUSSION TITLE
Changing the path of a lot of pages

### DISCUSSION THREAD

**Discussion by rutrum** [requarks/wiki#3464] [Help / Questions]
**Question**
Suppose I wanted one of the "folder" names in the root directory.  Do I have to individually change all 100 pages within that "folder"?  Or is there a way to do it automatically?  If there is no way, can we add that?  I don't want to commit to writing a large number of pages if I end up wanting to reorganize them in the future.  If I have to individually load the "change path" prompt for all 100 pages and rerender them it is going to take a long, long time.

**Host Info (please complete the following information):**
OS: Docker
Wiki.js version: 2.0.1
Database engine: postgres 9.7

**Comment by anton-yurchenko** [comment]
Doing that from a storage side (git) and syncing back results in duplication of a page.
I do not recommend doing it that way 😄 

_**P.S**: the fix was manually deleting the old files through an admin console._
